src/pages/aclmanagement.html

```diff
@@ -107,15 +107,16 @@ <h1 id="acl-management-view-title">
                                             {{rule.policy}}
                                         </span>
                                     </td>
-                                    <td class="role-cell data">
+                                    <td id="statementRoleCell" class="role-cell data">
```

Comment:
No. It's incorrect to have multiple elements with the same id on a page. Turn this to a css class instead!

Reply:
apply this to others as well